<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>73</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="Cam_Item">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="margin">
       <number>9</number>
      </property>
      <item>
       <widget class="QPushButton" name="Cam_Del_BTN">
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:red</string>
        </property>
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Cam_Num_Label">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>序号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="Cam_Con_CB">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>开启约束</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Cam_LE">
        <property name="placeholderText">
         <string>镜头名称</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
